tests-e2e/fixture/fixture.go

```diff
@@ -425,71 +425,71 @@ func EnsureDestinationNamespaceExists(namespaceParam string, argoCDNamespacePara
 	// Create the namespace again
 	_, err = kubeClientSet.CoreV1().Namespaces().Create(context.Background(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{
 		Name: namespaceParam,
-		Labels: map[string]string{
-			"argocd.argoproj.io/managed-by": argoCDNamespaceParam,
-		},
+		// Labels: map[string]string{
+		// 	"argocd.argoproj.io/managed-by": argoCDNamespaceParam,
+		// },
 	}}, metav1.CreateOptions{})
 	if err != nil {
-		return err
-	}
-
-	// Wait for Argo CD to process the namespace, before we exit:
-	// - This helps us avoid a race condition where the namespace is created, but Argo CD has not yet
-	//   set up proper roles for it.
-
-	if err := wait.PollImmediate(time.Second*1, time.Minute*2, func() (done bool, err error) {
-		var roleBindings *rbacv1.RoleBindingList
-		roleBindings, err = kubeClientSet.RbacV1().RoleBindings(namespaceParam).List(context.Background(), metav1.ListOptions{})
-		if err != nil {
-			return false, err
-		}
-
-		count := 0
-		// Exit the poll loop when there exists at least one rolebinding containing 'gitops-service-argocd' in it's name
-		// This helps us avoid a race condition where the namespace is created, but Argo CD has not yet
-		// set up proper roles for it.
-		for _, item := range roleBindings.Items {
-			if strings.Contains(item.Name, argoCDNamespaceParam+"-") {
-				count++
-			}
-		}
-
-		if count >= 2 {
-			// We expect at least 2 rolebindings
-			return true, nil
-		}
-
-		return false, nil
-	}); err != nil {
-		return err
-	}
-
-	if err := wait.PollImmediate(time.Second*1, time.Minute*2, func() (done bool, err error) {
-		var roles *rbacv1.RoleList
-		roles, err = kubeClientSet.RbacV1().Roles(namespaceParam).List(context.Background(), metav1.ListOptions{})
-		if err != nil {
-			return false, err
-		}
-
-		count := 0
-		// Exit the poll loop when there exists at least one rolebinding containing 'gitops-service-argocd' in it's name
-		// This helps us avoid a race condition where the namespace is created, but Argo CD has not yet
-		// set up proper roles for it.
-		for _, item := range roles.Items {
-			if strings.Contains(item.Name, argoCDNamespaceParam+"-") {
-				count++
-			}
-		}
-
-		if count >= 2 {
-			// We expect at least 2 roles
-			return true, nil
-		}
-
-		return false, nil
-	}); err != nil {
-		return fmt.Errorf("argo CD never setup rolebindings for namespace '%s': %v", namespaceParam, err)
-	}
+		return fmt.Errorf("unable to create namespace '%s': %v", namespaceParam, err)
```

Comment:
Rather than commenting these out (or deleting them), lets move them into their own function. We don't need to call that function, but we can at least have the function lying around if we need this functionality again.